.atomist/editors/AddTypeScriptEditor.ts

```diff
@@ -49,10 +49,14 @@ class AddTypeScriptEditor implements EditProject {
     description: string;
 
     edit(project: Project) {
-        if (!IsRugArchive(project) || !IsSetUpForTypeScript(project)) {
+
+        if (!IsRugArchive(project)) {
+            console.log("Can't run when not a rug archive");
             return;
         }
 
+        project.editWith("AddTypeScript", {});
```

Comment:
I think this is a good idea, very strongly.
Why would you not?

Why fail when we can implement the prerequisites for them?